internal/server/metrics.go

```diff
@@ -79,25 +80,11 @@ var (
 		[]string{"org", "user"},
 	)
 
-	totalMinutesUsedUbuntuActions = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "actions_total_minutes_used_ubuntu_minutes",
-		Help: "Total minutes used for Ubuntu type for the GitHub Actions.",
-	},
-		[]string{"org", "user"},
-	)
-
-	totalMinutesUsedMacOSActions = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "actions_total_minutes_used_macos_minutes",
-		Help: "Total minutes used for MacOS type for the GitHub Actions.",
-	},
-		[]string{"org", "user"},
-	)
-
-	totalMinutesUsedWindowsActions = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "actions_total_minutes_used_windows_minutes",
-		Help: "Total minutes used for Windows type for the GitHub Actions.",
+	totalMinutesUsedByRunnersActions = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "actions_total_minutes_used_by_runner_minutes",
```

Comment:
```suggestion
		Name: "actions_minutes_used_total",
```
Fits the Prometheus naming guidelines better I think.

Reply:
I'll change, thing is, we already have a `actions_total_minutes_used_minutes`, not sure if this will not cause some confusion since the naming is similar.